Mathlib/Topology/Algebra/InfiniteSum/Basic.lean

```diff
@@ -132,6 +132,10 @@ protected theorem Set.Finite.multipliable {s : Set β} (hs : s.Finite) (f : β 
 theorem multipliable_of_finite_mulSupport (h : (mulSupport f).Finite) : Multipliable f := by
   apply multipliable_of_ne_finset_one (s := h.toFinset); simp
 
+@[to_additive]
+lemma Finite.multipliable [Finite β] (f : β → α) : Multipliable f :=
```

Comment:
I think
```suggestion
lemma Multipliable.of_finite [Finite β] {f : β → α} : Multipliable f :=
```
is more useful in that it lets you write `some_theorem .of_finite` to get rid of the side condition.